coherence/session.go

```diff
@@ -504,6 +607,11 @@ func (s *Session) ensureConnection() error {
 	return nil
 }
 
+// IsGrpcV1 indicates if we are connected to a v1 gRPC client.
+func (s *Session) IsGrpcV1() bool {
```

Comment:
I think a more general call would be better here.  The protocol version could rev in the future.  Would it be better to have an accessor for the protocol version vs a one with the protocol version hard-coded?